src/node.rb

```diff
@@ -185,6 +185,8 @@ def build_method
 
     def build_method_class
       case repo_config[:build_method]&.to_sym
+      when :uefi
```

Comment:
From https://github.com/alces-software/metalware/issues/164#issuecomment-322753763 Steve says this is primarily a Kickstart build method. Unless you've heard differently since then I'd probably call this `uefi-kickstart`.

Reply:
`uefi` has been renamed in https://github.com/alces-software/metalware/pull/176/commits/67406f0b158d5702f3ce62c7b5cc8d18c1412ecd